packages/service-library/src/servicelib/fastapi/tracing.py

```diff
@@ -13,9 +13,44 @@
 from opentelemetry.sdk.resources import Resource
 from opentelemetry.sdk.trace import TracerProvider
 from opentelemetry.sdk.trace.export import BatchSpanProcessor
+from servicelib.logging_utils import log_context
 from settings_library.tracing import TracingSettings
 
-log = logging.getLogger(__name__)
+_logger = logging.getLogger(__name__)
+#########
+try:
+    from opentelemetry.instrumentation.asyncpg import AsyncPGInstrumentor
+
+    HAS_ASYNCPG = True
+except ImportError:
+    HAS_ASYNCPG = False
+#########
```

Comment:
MINOR: there is no need for these `### ` comments

Reply:
done